### DISCUSSION TITLE
Welcome to CppMicroServices Discussions!

### DISCUSSION THREAD

**Discussion by jeffdiclemente** [CppMicroServices/CppMicroServices#516] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kawshol** [comment]
Hello Jeff,

We are planning to use the cppmicrosrevice with windows IoT core for our gateway on our board.

Before we use it we would like to test this on windows 10 . 

Would you be able to give our engineer some instrction on how to build the code in windows and test it so that we can take it further.

This will be used in automotive industry in car for specific application. Though documentation is availble, It will be a great help if you give us simple instructions to do the first test.

We will then take a deeper look to check what more is required to enhance.

I appriciate your help, 

Best Regards,
Kawshol Sharma 
kawshol@devanture.org 
+6596515140

**Reply by jeffdiclemente** [reply]
Hi Kawshol,

I recommend following the [build instructions](http://docs.cppmicroservices.org/en/stable/doc/src/build_instructions.html). You will want to have CMake generate a Visual Studio solution and I recommend leaving all the CMake configuration options as default except for `US_BUILD_TESTING` which you will need to enable to run all the unit tests. 

Once CMake has generated a Visual Studio solution you want to run the `ALL_BUILD` project to build everything and then build the `RUN_TESTS` project to run all the unit tests.

I also recommend using the latest stable version of CMake and Visual Studio.

Hope that helps.

Thanks,
Jeff

**Comment by kawshol** [comment]
Hello Jeff,

Thanks for the help. we are on our way to put this under the windows IOT
core.

/Kwashol

On Mon, Dec 14, 2020 at 8:29 PM Jeff DiClemente <notifications@github.com>
wrote:

> Hi Kawshol,
>
> I recommend following the build instructions
> <http://docs.cppmicroservices.org/en/stable/doc/src/build_instructions.html>.
> You will want to have CMake generate a Visual Studio solution and I
> recommend leaving all the CMake configuration options as default except for
> US_BUILD_TESTING which you will need to enable to run all the unit tests.
>
> Once CMake has generated a Visual Studio solution you want to run the
> ALL_BUILD project to build everything and then build the RUN_TESTS
> project to run all the unit tests.
>
> I also recommend using the latest stable version of CMake and Visual
> Studio.
>
> Hope that helps.
>
> Thanks,
> Jeff
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/CppMicroServices/CppMicroServices/discussions/516#discussioncomment-204685>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABIJVRJRV6LOZLJSE4WBC3SUX3O7ANCNFSM4UU5EODQ>
> .
>


-- 

Best Regards,
Kawshol Sharma
+65 96515140
